docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,60 @@
+# Introducing the new Azure SDK Management Libraries for JS
+
+We're excited to announce the new GA release for Javascript SDKs. These SDKs have several new features that help our customers to have a great experience in using the Azure Services. Some of the important SDKs that have been released are:
+
+1. [Compute](https://www.npmjs.com/package/@azure/arm-compute/v/17.0.0)
+2. [Network](https://www.npmjs.com/package/@azure/arm-network/v/26.0.0)
+3. [Storage](https://www.npmjs.com/package/@azure/arm-storage/v/17.0.0)
+4. [Sql](https://www.npmjs.com/package/@azure/arm-sql/v/9.0.0)
+5. [Resources](https://www.npmjs.com/package/@azure/arm-resources/v/5.0.0)
+6. [Policy](https://www.npmjs.com/package/@azure/arm-policy/v/5.0.0)
+7. [Managed applications](https://www.npmjs.com/package/@azure/arm-managedapplications/v/2.0.0)
+8. [Links](https://www.npmjs.com/package/@azure/arm-links/v/2.0.0)
+9. [Locks](https://www.npmjs.com/package/@azure/arm-locks/v/2.0.0)
+10. [Features](https://www.npmjs.com/package/@azure/arm-features/v/3.0.0)
+
+Each of these SDKs follows the new [Azure SDK guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html). All these SDKs have been generated using the latest version of the new [Autorest Typescript SDK Generator](https://www.npmjs.com/package/@autorest/typescript/v/6.0.0-beta.15). These libraries provide new authentication mechanisms, uses Promises, etc. We encourage you to try out the libraries and provide your feedback.
+
+## Key concepts
+
+The new libraries have the following changes:
+
+1. **Authentication**: The packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are replaced with a new package `@azure/identity` instead. The [migration guide](https://github.com/Azure/ms-rest-nodeauth/blob/master/migrate-to-identity-v2.md) helps in the process of migration to the new identity package.
+2. **Promises**: Method overloads that use callbacks have been replaced to use Promise instead.
+3. The results of List operations could be iterated by using the `PagedAsyncIterableIterator` interface
```

Comment:
Can we add something about why this is a better experience?

Reply:
Added it. Let me know if it works